libs/payments/capability/src/lib/capability.manager.spec.ts

```diff
@@ -5,58 +5,66 @@ import { Test, TestingModule } from '@nestjs/testing';
 
 import {
   ContentfulManager,
+  CapabilityCapabilitiesResultFactory,
+  CapabilityOfferingResultFactory,
+  CapabilityServiceByPlanIdsResultUtil,
+  CapabilityServicesResultFactory,
   ServiceResultFactory,
   ServicesWithCapabilitiesResultUtil,
 } from '@fxa/shared/contentful';
 import { CapabilityManager } from './capability.manager';
 
 describe('CapabilityManager', () => {
-  let manager: CapabilityManager;
-  let mockContentfulManager: ContentfulManager;
-  let mockResult: ServicesWithCapabilitiesResultUtil;
-
-  beforeEach(async () => {
-    mockResult = {} as any;
-    mockContentfulManager = {
-      getServicesWithCapabilities: jest.fn().mockResolvedValueOnce(mockResult),
-    } as any;
-
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [
-        { provide: ContentfulManager, useValue: mockContentfulManager },
-        CapabilityManager,
-      ],
-    }).compile();
-
-    manager = module.get<CapabilityManager>(CapabilityManager);
-  });
+  describe('getClients', () => {
+    let manager: CapabilityManager;
+    let mockContentfulManager: ContentfulManager;
+    let mockResult: ServicesWithCapabilitiesResultUtil;
 
-  it('should be defined', async () => {
-    expect(manager).toBeDefined();
-    expect(manager).toBeInstanceOf(CapabilityManager);
-  });
+    beforeEach(async () => {
+      mockResult = {} as any;
```

Comment:
Non-blocking since you did not add, but a note - would be nice for this not to be cast to any and to properly implement this mock util here.